src/RoatpProviderModeration/SFA.DAS.RoatpProviderModeration.Application/Queries/GetProvider/GetProviderQuery.cs

```diff
@@ -0,0 +1,14 @@
+﻿using MediatR;
+
+namespace SFA.DAS.RoatpProviderModeration.Application.Queries.GetProvider
+{
+    public class GetProviderQuery : IRequest<GetProviderQueryResponse>
+    {
+        public int Ukprn { get; set; }
+    }
+
+    public class GetProviderQueryResponse
+    {
+
```

Comment:
empty space

Reply:
Will sort it when it gets implemented... just a place holder for now